src/main/kotlin/step4/racingcar/racingcar/GameResult.kt

```diff
@@ -0,0 +1,12 @@
+package step4.racingcar.racingcar
+
+data class GameResult(
+    val cars: List<Car>,
+    val round: Int
+) {
+    fun getWinner(): String {
```

Comment:
```suggestion
    fun getWinners(): String {
```

우승자가 한 명 일 수도 있고 여러 명 일 수도 있기 때문에, 복수형이 더 낫지 않을까요?